test-reqs.txt

```diff
@@ -1,20 +1,20 @@
+alabaster
```

Comment:
As noted in #4120, most of these are already installed as part of the standard installation and don't actually need to be here. The only things that _should_ be here are those listed in the `test` extras section.

Reply:
+1; I prefer to #4120. Could you revert this please?